app/models/card.rb

```diff
@@ -7,6 +7,12 @@ def validate(record)
 end
 
 class Card < ActiveRecord::Base
+	before_validation :define_review_date
 	validates :original_text,:translated_text, presence: true
 	validates_with CompareValidator
+
+    protected
```

Comment:
Indent access modifiers like `protected`.<br>Keep a blank line before and after `protected`.